README.md

```diff
@@ -8,13 +8,85 @@ This is the code for the [shescoding.org](http://shescoding.org) website.
 
 **Open-source?**
 
-Yes! *She's Coding* is build by the community, for the community. We currently have a small group of volunteers bootstrapping the project, and we are in the process of defining some simple rules on how to contribute for the wider community.
+Yes! *She's Coding* is built by the community, for the community. We currently have a small group of volunteers bootstrapping the project, and we are in the process of defining some simple rules on how to contribute for the wider community.
 
 [Let us know](mailto:info@shescoding.org) if you want to join our team!
 
 ### Installation
 
-+ Install [Homebrew](http://brew.sh/) (OSX):
++ **Ubuntu** for Windows 10 Users
```

Comment:
since we also have just Linux, maybe it's best to have a special case Windows 10 Users section?
when it's inline with the rest, it's very hard to differentiate which one is which.

Reply:
I'll change the README to incorporate a Windows 10 section.